source/Assets/Plugins/Didomi/IOS/Didomi.mm

```diff
@@ -305,9 +305,48 @@ bool getUserConsentStatusForVendorAndRequiredPurposes(char* vendorId)
 
 	    }
 
+	int getUserLegitimateInterestStatusForPurpose(char* purposeId)
 
 
 
+	{
```

Comment:
there seems to be something odd with the format in this file, is there a reason to keep 3 blank lines between each line of code?

Reply:
Yes, fixed.